sw/dnn/fused_concat_linear/data/params.json

```diff
@@ -7,5 +7,6 @@
     input_shape: [32, 4],
     output_shape: [32, 16],
     dtype: "FP64",
-    baseline: true
+    baseline: true,
```

Comment:
The `baseline` parameter should not be needed anymore.

Reply:
Addressed in [this ](https://github.com/pulp-platform/snitch_cluster/pull/137/commits/9766b87c6e9cdf9d656117b7b18bc0a4ef78d71f)commit.